packages/twenty-server/.eslintrc.cjs

```diff
@@ -13,6 +13,7 @@ module.exports = {
         project: ['packages/twenty-server/tsconfig.json'],
       },
       rules: {
+        'lingui/t-call-in-function': 'off',
```

Comment:
In most cases the lingui helper replaces t() by a function call to immediately translate the string. In our case, we have to store the value in DB first and then do the translation later, so the t flag is not there to translate but only to indicate that a string should be added to the translation catalog

Reply:
You'd better to use `msg` macro in that case. It's made exactly for that purpose. To mark message for translation but not translate it right away.

Reply:
Oh nice thanks a lot, will update!